crates/driver/src/infra/config/file/mod.rs

```diff
@@ -574,3 +577,61 @@ pub enum GasEstimatorType {
     Native,
     Web3,
 }
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct OrderPriorityConfig {
+    /// Defines order prioritization strategies that will be applied in the
+    /// specified order.
+    #[serde(default = "default_order_priority_strategies")]
+    pub strategies: Vec<OrderPriorityStrategy>,
+
+    /// Configures the threshold for `CreationTimestamp` order prioritization
+    /// strategy. Orders created within this threshold will be prioritized.
+    /// Takes effect only if `CreationTimestamp` is specified in
+    /// `order_priority_strategies`.
+    #[serde(default = "default_order_creation_timestamp_threshold")]
+    pub order_creation_timestamp_threshold: Duration,
+}
+
+impl Default for OrderPriorityConfig {
+    fn default() -> Self {
+        OrderPriorityConfig {
+            strategies: default_order_priority_strategies(),
+            order_creation_timestamp_threshold: default_order_creation_timestamp_threshold(),
+        }
+    }
+}
+
+/// Defines various strategies to prioritize orders.
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub enum OrderPriorityStrategy {
+    /// Strategy to prioritize orders based on external price.
+    /// This strategy uses the likelihood that an order will be fulfilled,
+    /// based on token prices. A larger value means that the order is more
+    /// likely to be fulfilled.
+    ExternalPrice,
```

Comment:
Why is the order class strategy not here? Bit confusing that some parts are in a config and one is hidden as a default thing in the code.